sunpy/visualization/animator/image.py

```diff
@@ -140,6 +138,8 @@ class ImageAnimatorWCS(ImageAnimator):
     ----------
```

Comment:
Does this not get inherited from the base class?

Reply:
The docstring? I would assume so, but we are adding a `wcs` parameter (that was previously not included in the docstring).